test/integration/template_database_test.go

```diff
@@ -2531,85 +2671,169 @@ func TestDatabaseStatefulSetVolumeSnapshot(t *testing.T) {
 			for _, template := range obj.Spec.VolumeClaimTemplates {
 				assert.Nil(t, template.Spec.DataSourceRef)
 			}
-			assert.False(t, testlib.EnvContains(obj.Spec.Template.Spec.Containers[0].Env, "SNAPSHOT_RESTORED", "true"))
+			assert.True(t, testlib.EnvContains(obj.Spec.Template.Spec.Containers[0].Env, "SNAPSHOT_RESTORED", "false"))
```

Comment:
Was that a bug?

Reply:
Before it was checking for the absence of the env variable. Now it is explicitly set to false.